README.md

```diff
@@ -270,6 +270,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Runs Lighthouse and posts results to PRs and Slack](https://github.com/foo-software/lighthouse-check-action)
 - [Run Lighthouse in CI using GitHub Actions](https://github.com/treosh/lighthouse-ci-action)
 - [Continuous Benchmarking and Benchmark Visualization for Go](https://github.com/bobheadxi/gobenchdata)
+- [Size Limit Action](https://github.com/andresz1/size-limit-action) - Compares the real cost of your JS in every PR and rejects it if exceeds the limit using [Size Limit](https://github.com/ai/size-limit)
```

Comment:
Since most of the actions on this page are dependent on a different modules (i.e. Size Limit), it's preferable that you leave that out of the description for brevity sake. Also, for clarity in the description, I'd possibly change it to something along the lines of "Comments cost comparison of your JS in PR's and rejects them if limit is exceeded."? Maybe I'm losing context with that description, but if I were just browsing this list, I wouldn't know what "real cost" is.

Reply:
Thank you for the feedback! I agree, changes made!